pkg/resources/management.cattle.io/v3/globalrolebinding/validator_test.go

```diff
@@ -672,6 +837,11 @@ func TestAdmit(t *testing.T) {
 			}
 			require.NoError(t, err)
 			require.Equalf(t, test.allowed, response.Allowed, "Response was incorrectly validated wanted response.Allowed = '%v' got '%v' message=%+v", test.allowed, response.Allowed, response.Result)
+			if test.respMessageContains != nil {
```

Comment:
I would not recommend validating on the message. The message isn't part of the interface, it can contain an arbitrary string which is meaningful to the user, but from a code-perspective there's no guarantee on what is in that string.